PsfRuntime/Config.cpp

```diff
@@ -280,16 +280,12 @@ void load_json()
                         {
                             Log("Found config at: %ls", dentry.path().c_str());
 #pragma warning(suppress:4996) // Nonsense warning; _wfopen is perfectly safe
-                            file = _wfopen(dentry.path().c_str(), L"rb, ccs=UTF-8");
+                            THROW_IF_FAILED(file = _wfopen(dentry.path().c_str(), L"rb, ccs=UTF-8"));

```

Comment:
Should be:
file = _wfopen(dentry.path().c_str(), L"rb, ccs=UTF-8")
THROW_HR_IF(HRESULT_FROM_WIN32(ERROR_OPEN_FAILED), !file)

Reply:
resolved

---
In reply to: [362677740](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/119#discussion_r362677740) [](ancestors = 362677740)